pages/linux/pacman4console.md

```diff
@@ -0,0 +1,20 @@
+# pacman4console
+
+> A text-based console game inspired by the original Pacman.
+> You play as Pacman (C); avoid the ghosts (&'s); collect all the pellets (*'s and .'s).
+
+- Start a game (at level 1 by default):
+
+`pacman4console`
+
+- Start a game on a certain level (there are nine official levels):
+
+`pacman4console --level={{#}}`
+
+- Start the pacman4console editor with a level text-file:
+
+`pacmanforconsoleedit {{level_file}}`
+
+- Start a non-standard level (like a custom one):
```

Comment:
Perhaps this will read better?
```suggestion
- Play a custom level:
```

Also, could `level_file` on the line below be `path/to/level_file` please? Then it'll be more consistent with our existing pages :-)